Documentation/community/roadmap.rst

```diff
@@ -14,35 +14,60 @@ broader community. You'll also find here some pointers on how you can
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`bandwidth-manager`                        | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+| Cilium Mesh                                      | Stable (:ref:`Roadmap Details<rm-clustermesh>`)          |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Multi-Cluster (ClusterMesh)<clustermesh>` | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`External Workloads<external_workloads>`   | Beta                                                     |
+++-------------------------------------------------+----------------------------------------------------------+
+| Hubble Observability                             | Stable (:ref:`Roadmap Details<rm-hubble-observability>`) |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Hubble CLI<hubble_cli>`                   | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`hubble_ui`                                | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Prometheus metrics<metrics>`              | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+| Cilium Service Mesh                              | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)  |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_ingress`                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_gateway_api`                           | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_l7_traffic_management`                 | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gs_mutual_authentication`                 | Beta                                                     |
```

Comment:
This one says `Mutual Authentication (Beta) | Beta` so “Beta” appears twice - although it probably doesn't matter.

Reply:
I think fixing this would mean re-sizing the table (because RST is so awesome!) and I don't think it's worth the effort